e2e/tests/example.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from "@playwright/test";
+
+test("has title", async ({ page }) => {
+  await page.goto("/");
+
+  await expect(page).toHaveTitle(
+    "Klaw | Kafka Self-service Topic Management Portal",
+  );
+});
+
+test("has login form", async ({ page }) => {
```

Comment:
Seeing this test makes me curious: would the e2e test environment allow for logging in? Was it tried? ^^

Reply:
Yes, it would, with the superadmin account - bc. that user is already there. There is no user and other date (yet) for other tests of course.